src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/upgrade/upgrade.component.html

```diff
@@ -197,6 +200,9 @@ <h5>
       <i [ngClass]="[icons.danger]"></i>
       {{ errorMessage }}
     </span>
+    <a class="mt-2 link-primary mb-2"
```

Comment:
There is already a margin-bottom for the above span. And adding another margin-top to this `a` makes it look like there are a lot of space in between. probably just the `mb-2` is enough here?
```suggestion
    <a class="link-primary mb-2"
```